From: Georges Basile Stavracas Neto <georges.stavracas@gmail.com>
Date: Tue, 31 Mar 2020 21:58:44 +0000
Subject: tests/actor-pick: Remove tabs

They're evil.

Forwarded: https://gitlab.gnome.org/GNOME/mutter/-/merge_requests/1169
Applied-upstream: 3.36.2, commit:31809e121424ec6609e947ab2d6bde6ae792b539
---
 src/tests/clutter/conform/actor-pick.c | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/src/tests/clutter/conform/actor-pick.c b/src/tests/clutter/conform/actor-pick.c
index 09c5bf7..53ac8ec 100644
--- a/src/tests/clutter/conform/actor-pick.c
+++ b/src/tests/clutter/conform/actor-pick.c
@@ -175,10 +175,10 @@ actor_pick (void)
   for (y = 0; y < ACTORS_Y; y++)
     for (x = 0; x < ACTORS_X; x++)
       {
-	ClutterColor color = { x * 255 / (ACTORS_X - 1),
-			       y * 255 / (ACTORS_Y - 1),
-			       128, 255 };
-	ClutterActor *rect = clutter_rectangle_new_with_color (&color);
+        ClutterColor color = { x * 255 / (ACTORS_X - 1),
+                               y * 255 / (ACTORS_Y - 1),
+                               128, 255 };
+        ClutterActor *rect = clutter_rectangle_new_with_color (&color);
 
         clutter_actor_set_position (rect,
                                     x * state.actor_width,
@@ -187,9 +187,9 @@ actor_pick (void)
                                 state.actor_width,
                                 state.actor_height);
 
-	clutter_actor_add_child (state.stage, rect);
+        clutter_actor_add_child (state.stage, rect);
 
-	state.actors[y * ACTORS_X + x] = rect;
+        state.actors[y * ACTORS_X + x] = rect;
       }
 
   clutter_actor_show (state.stage);
